Survival/Assets/Scripts/AnimalManager.cs

```diff
@@ -25,7 +25,7 @@ void Start()
 
     void Update()
     {
-        transform.rotation = Quaternion.Euler(0f,0f,rotation);
+        transform.rotation = Quaternion.Euler(0f, 0f, rotation);
```

Comment:
wow really gonna do me like that

Reply:
I think it was my prettier doing its work.